source/patterns/@aws-solutions-constructs/aws-events-rule-kinesisstreams/test/integ.events-rule-kinesisstreams-existing-eventbridge.ts

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
existing-eventbus, not existing-eventbridge?

Reply:
fixed them and updated the PR